schema/common/validation/configuration/rook.yml

```diff
@@ -0,0 +1,7 @@
+"$id": "#/specification"
```

Comment:
same as above

Reply:
I will add optional validation, but won't specify anything here.

Reply:
This doesn't make sense anyway as this will be taken from defaults, so I will leave this as it is.